manifester/settings.py

```diff
@@ -16,6 +16,7 @@
 validators = [
     # Validator("offline_token", must_exist=True),
     Validator("simple_content_access", default="enabled"),
+    Validator("username_prefix", len_min=3),
```

Comment:
what is purpose of this parameter `len_min=3` ?

Reply:
The username prefix has to be of minimum of 3 characters!

Reply:
@vijaysawant Jitendra is correct. The username_prefix will be used to identify which user created a subscription allocation when multiple users are using a shared RHSM account. Although it's ultimately up to the user to decide the value for username_prefix, my thinking was that first initial + last name would be the typical format. Three characters seemed like a reasonable minimum for this format. 

So the parameter that you highlighted will cause Dynaconf to throw an error when it evaluates the settings at runtime if the value of the username_prefix setting is less than 3 characters.